microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks.md

```diff
@@ -25,8 +25,6 @@ As explained in [Fix memory problems](index.md), memory issues affect page perfo
 <!-- ====================================================================== -->
 ## Open the Detached Elements tool
```

Comment:
I am investigating & stating for the record: Does the Experimental Features article need to be updated?

Reply:
Thanks Michael for thinking about updating the experimental features page!